timeserio/utils/pickle.py

```diff
@@ -1,31 +1,18 @@
-import s3fs
+import tentaclio as tio
 import joblib.externals.cloudpickle as cp
 
 __all__ = ['dumpf', 'loadf']
 
 
-def open_url(filename, mode):
```

Comment:
if I remember correctly, this exact function was the motivation to write tentaclio

Reply:
the circle has been completed

Reply:
ahaaaaa yep